src/ScrumBoardItBundle/Services/Api/ApiJira.php

```diff
@@ -51,23 +51,29 @@ private function getIssues($data)
     {
         $issues = array();
         foreach ($data->issues as $issue) {
-            if ($issue->fields->issuetype->subtask) {
-                $task = new SubTask();
-            } else {
-                $task = new Task();
-                if (! empty($issue->fields->{$this->config['complexity_field']})) {
+            $task = new Task();
+            switch($issue->fields->issuetype->name) {
+                case 'Récit':
                     $task->setComplexity($issue->fields->{$this->config['complexity_field']});
                     $task->setUserStory(true);
-                } else {
+                    break;
+                case 'Sous-tâche':
```

Comment:
It depends of the labels defined by Jira... It is not universal, I admit it's a problem